plugins/namelayer-paper/src/main/java/vg/civcraft/mc/namelayer/group/Group.java

```diff
@@ -105,15 +107,38 @@ public List<UUID> getAllMembers() {
 	 * @return Returns all the UUIDS of the specific PlayerType.
 	 */
 	public List<UUID> getAllMembers(PlayerType type) {
-		List<UUID> uuids = Lists.newArrayList();;
-		for (Map.Entry<UUID, PlayerType> entry : players.entrySet()) {
+		List<UUID> uuids = Lists.newArrayList();
+        for (Map.Entry<UUID, PlayerType> entry : players.entrySet()) {
 			if (entry.getValue() == type) {
 				uuids.add(entry.getKey());
 			}
 		}
 		return uuids;
 	}
-	
+
+	/**
+	 * Returns all the uuids of the invitees in this group.
+	 * @return Returns all the uuids.
+	 */
+	public List<UUID> getAllInvites() {
+		return Lists.newArrayList(invites.keySet());
```

Comment:
this function is a throwback from the java 7 days, you can do `new ArrayList<>(invites.keySet())` now